cartography/intel/kubernetes/rbac.py

```diff
@@ -0,0 +1,439 @@
+            if subject.kind == "ServiceAccount"
+        ]
+
+        for subject in service_account_subjects:
+            result.append(
+                {
+                    "id": f"{rb.metadata.namespace}/{rb.metadata.name}/{subject.namespace}/{subject.name}",  # same role binding can be used for different service accounts so need a unique id
+                    "name": rb.metadata.name,
+                    "namespace": rb.metadata.namespace,
+                    "uid": rb.metadata.uid,
+                    "creation_timestamp": get_epoch(rb.metadata.creation_timestamp),
+                    "resource_version": rb.metadata.resource_version,
+                    "role_name": rb.role_ref.name,
+                    "role_kind": rb.role_ref.kind,
+                    "subject_name": subject.name,
+                    "subject_namespace": subject.namespace,
+                    "subject_service_account_id": f"{subject.namespace}/{subject.name}",
+                    "role_id": f"{rb.metadata.namespace}/{rb.role_ref.name}",
+                }
+            )
+    return result
+
+
+def transform_cluster_roles(cluster_roles: List[V1ClusterRole]) -> List[Dict[str, Any]]:
+    """
+    Transform Kubernetes ClusterRoles into a list of dictionaries.
+    Flattens rules into separate api_groups, resources, and verbs lists.
+    """
+    result = []
+    for cluster_role in cluster_roles:
+        # Flatten all rules into combined lists
+        all_api_groups = []
+        all_resources = []
+        all_verbs = []
+
+        for rule in cluster_role.rules or []:
+            # Extend api_groups, handling None and empty string cases
+            if rule.api_groups:
+                for api_group in rule.api_groups:
+                    # Empty string represents core API group
+                    api_group_name = "core" if api_group == "" else api_group
+                    if api_group_name not in all_api_groups:
+                        all_api_groups.append(api_group_name)
+
+            # Extend resources
+            if rule.resources:
+                for resource in rule.resources:
+                    if resource not in all_resources:
+                        all_resources.append(resource)
+
+            # Extend verbs
+            if rule.verbs:
+                for verb in rule.verbs:
+                    if verb not in all_verbs:
+                        all_verbs.append(verb)
+
+        result.append(
+            {
+                "id": cluster_role.metadata.name,
+                "name": cluster_role.metadata.name,
+                "uid": cluster_role.metadata.uid,
+                "creation_timestamp": get_epoch(
+                    cluster_role.metadata.creation_timestamp
+                ),
+                "resource_version": cluster_role.metadata.resource_version,
+                "api_groups": all_api_groups,
+                "resources": all_resources,
+                "verbs": all_verbs,
```

Comment:
```suggestion
                "api_groups": list(all_api_groups),
                "resources": list(all_resources),
                "verbs": list(all_verbs),
```
Im not sure whether this is required for the neo4j driver, perhaps it supports sets too? But if it doesn't then you'll have to cast to list